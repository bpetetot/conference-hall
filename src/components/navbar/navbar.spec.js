/* eslint-env jest */
import snap from 'tests/snapshot'
import NavBar from './index.js'

const snapshot = props => snap(NavBar)({ ...props })

describe('components/navbar', () => {
  it('should render', snapshot({ className: 'c', children: 'children' }))
})
