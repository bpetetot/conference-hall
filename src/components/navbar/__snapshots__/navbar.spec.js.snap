// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/navbar should render 1`] = `
<nav
  className="navbar c"
>
  children
</nav>
`;
